backend/src/main/java/com/bakdata/conquery/models/index/IndexService.java

```diff
@@ -87,7 +87,7 @@ public Index<?> load(@NotNull IndexKey<?> key) throws Exception {
 				}
 			}
 			catch (IOException ioException) {
-				log.warn("Failed to open `{}`", key.getCsv(), ioException);
+				log.warn("Failed to open `{}`: {} (enable TRACE for exception)", key.getCsv(), ioException, (Exception) (log.isTraceEnabled() ? ioException : null));
```

Comment:
warum möchtest du die Exception schlucken? Das gibt schon gute Info

Reply:
Ich hatte lokal getestet und da keine Referenzen gehabt weshalb die logs voll mit Stacktraces waren.
Mit der Log Message wird nur der Key (der den Pfad enthält) und der Name der Exception, z.B. `FileNotFoundException` geprintet.
Ich denke das reicht hier in den meisten Fällen schon aus um die Ursache zu finden.